source/agora/node/Validator.d

```diff
@@ -597,6 +620,6 @@ public class Validator : FullNode, API
         // Network needs Validators, see if we can enroll
         if (this.config.validator.recurring_enrollment &&
             msg == Ledger.InvalidConsensusDataReason.NotEnoughValidators)
-            this.checkAndEnroll(this.ledger.getBlockHeight());
+            this.checkAndEnroll(this.ledger.getBlockHeight(), true);
```

Comment:
Can you explain why this calling has `true` for `dont_wait`?

Reply:
The case here is that we don't have enough validators for the next round. To get out of that situation faster we ignore the timeout.